src/ProfilesList.json

```
@@ -250,5 +250,6 @@
   "upendra.json",
   "Kritika30032002.json",
   "dellucifer.json",
-  "shricodev.json"
-]
\ No newline at end of file
+  "shricodev.json",
+  "pratikkumar399"
```

```suggestion
  "pratikkumar399.json"
```
Add full filename that is `pratikkumar399.json`